conda/environments/builddocs_py35.yml

```diff
@@ -17,8 +17,8 @@ dependencies:
 - nvstrings
 - pandas=0.20.*
 - pyarrow=0.10.*
-- pip:
-  - numpydoc
-  - sphinx
-  - sphinx-rtd-theme
-  - sphinxcontrib-websupport
```

Comment:
@mike-wendt unsure if you'd know this but seeing as you've fought with RTD the most would moving these away from pip to conda be okay?